docs/source/overview.md

```diff
@@ -0,0 +1,77 @@
+# SimPhoNy: interoperability powered by ontology
+
+SimPhoNy is an ontology-based framework aimed at enabling interoperability between different simulation and data management tools, with a focus on materials science.
+
+
+
+## What can SimPhoNy be used for?
+
+### Manipulate ontology-based linked data, a format well suited for [FAIR data principles](https://en.wikipedia.org/wiki/FAIR_data).
```

Comment:
Not sure if I would put a full stop at the end of each sentence, since they are headers